src/bh_route_vars.erl

```diff
@@ -13,22 +13,18 @@
 -define(S_VAR_LIST, "var_list").
 -define(S_VAR, "var_get").
 
--define(SELECT_VARS_BASE, "select v.name, v.type, v.value from vars_inventory v ").
-
 prepare_conn(Conn) ->
-    {ok, S1} = epgsql:parse(Conn, ?S_VAR_LIST,
-                            [?SELECT_VARS_BASE,
-                             "order by name"
-                            ], []),
-
-    {ok, S2} = epgsql:parse(Conn, ?S_VAR,
-                            [?SELECT_VARS_BASE,
-                             "where v.name = $1"
-                            ], []),
-
-    #{?S_VAR_LIST => S1,
-      ?S_VAR => S2
-     }.
+    PrivDir = code:priv_dir(blockchain_http),
```

Comment:
Can we pull this into a db_worker helper function and use in at least two places ?

Reply:
yeah good idea